test/Factories.hs

```diff
@@ -37,8 +37,8 @@ createUser ident = do
     Entity planId _ <- createFreePlan
 
     insertEntity User
-        { userName = "John Smith (" ++ ident ++ ")"
-        , userEmail = "john-" ++ ident ++ "@gmail.com"
+        { userName = "Dummy Login"
+        , userEmail = "dummy@example.com"
```

Comment:
Actually, returning `Left` here will keep the same semantics as before: dummy logins are only supported for existing users, the error will be discarded in that case.